internal/service/advancedclustertpf/common.go

```diff
@@ -13,6 +13,14 @@ import (
 	"go.mongodb.org/atlas-sdk/v20241113004/admin"
 )
 
+const (
+	IgnoreLabelKey = "Infrastructure Tool"
+)
+
+var (
+	ErrIgnoreLabel = fmt.Errorf("you should not set `%s` label, it is used for internal purposes", IgnoreLabelKey)
```

Comment:
```suggestion
	ErrIgnoreLabel = fmt.Errorf("`%s` label value is not supported as it is reserved for internal purposes", IgnoreLabelKey)
```
not strictly related to PR but thinking we could improve wording here

Reply:
@AgustinBettati can you tell your suggestion?

Reply:
I actually left it as a code suggestion 😄

Reply:
oops, my bad

Reply:
oops, my bad, changed with a small adjustment: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2978/commits/f2eca7c15fe7c5120353b6f7c0063bd12ae38436